src/box/lua/config/init.lua

```diff
@@ -578,6 +578,27 @@ function methods.instances(self)
     return res
 end
 
+function methods.instance_uri(self, uri_type, opts)
+    selfcheck(self, 'instance_uri')
+    uri_type = uri_type == nil and 'peer' or uri_type
+    if uri_type ~= 'peer' and uri_type ~= 'sharding' then
+        error(('Expected "peer" or "sharding", got %s'):format(uri_type), 0)
```

Comment:
Nit: "peer" and "sharding" is in quotes, so I would use `%q` instead of `%s` for consistency.

Reply:
Fixed.